README.md

```diff
@@ -84,7 +84,9 @@ Clone this repository to your desired folder:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **Pending**
+- [ ] **Home page**
+- [ ] **Drag and Drop**
+- [ ] **contact page**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Great job working on this milestone and adding all the sections of the README. Under the `Contributing` section, you are supposed to have a link to an issue so that users can make contributions. This link is missing at the moment. Kindly add it